build.sbt

```diff
@@ -0,0 +1,28 @@
+val appName = "address-lookup-performance-tests"
+val appVersion = "0.1.0-SNAPSHOT"
+
+lazy val root = (project in file("."))
+  .enablePlugins(GatlingPlugin)
+  .enablePlugins(CorePlugin)
+  .enablePlugins(JvmPlugin)
+  .enablePlugins(IvyPlugin)
```

Comment:
What do we use the CorePlugin, JVMPlugin and IvyPlugin for?  Seems to work fine without them.